api/v1beta2/ibmvpccluster_types.go

```diff
@@ -98,26 +111,108 @@ type AdditionalListenerSpec struct {
 	Port int64 `json:"port"`
 }
 
-// VPCNetworkSpec defines the desired state of the network resources for the cluster for extended VPC Infrastructure support.
-type VPCNetworkSpec struct {
-	// workerSubnets is a set of Subnet's which define the Worker subnets.
+// BackendPoolSpec defines the desired configuration of a VPC Load Balancer Backend Pool.
+type BackendPoolSpec struct {
+	// name defines the name of the Backend Pool.
 	// +optional
-	WorkerSubnets []Subnet `json:"workerSubnets,omitempty"`
+	Name *string `json:"name,omitempty"`
```

Comment:
lets add a regex for the validation here - `Possible values: 1 ≤ length ≤ 63, Value must match regular expression ^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$`